.github/workflows/firestore-rules-pull-request.yml

```
@@ -13,12 +14,8 @@ jobs:
     environment: staging
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-node@v4
-        with:
-          node-version: 22
-          cache: 'yarn'
-      - name: Install Firebase CLI
-        run: npm install -g firebase-tools
+      - name: CI setup action for Firebase deploy
+        uses: ./.github/actions/setup-firebase-deploy
```

actionlint

description is required in metadata of "Setup Node and install Firebase CLI" action at "/inmem/34/4e7789b2-7023-477b-9548-9da77b8115ae/home/jailuser/git/.github/actions/setup-firebase-deploy/action.yml"

(action)